pom.xml

```diff
@@ -98,6 +101,11 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
+                <groupId>org.springframework.cloud</groupId>
+                <artifactId>spring-cloud-skipper-autoconfigure</artifactId>
+                <version>1.0.2.BUILD-SNAPSHOT</version>
```

Comment:
WDYT about replacing the literal version string with `${project.version}` ?

Reply:
I don't remember why I went with actual version as at some point these were using `${project.version}`. Don't remember anymore if I had some issues with that. I had some many issues that I shuffled things around so much. We can try later and see if it works(it prob should be ok).